Orange/widgets/unsupervised/owdistancemap.py

```diff
@@ -441,21 +449,22 @@ def set_distances(self, matrix):
 
     def set_items(self, items, axis=1):
         self.items = items
-        model = self.annot_combo.model()
+        names = self.Names
         if items is None:
-            model[:] = ["None", "Enumeration"]
+            model = DomainModel(order=(names[self.Enumeration], ))
```

Comment:
Why is this a domain model? It doesn't have any variables, let alone domain, does it?